### ISSUE TITLE
refinement fraction=1 should do global refinement

### ISSUE THREAD

**Issue by tjhei** [geodynamics/aspect#3141]
seen while working on #3119

**Comment by bangerth** [comment]
Does it not? If so, this must be an issue in deal.II, but I'm really surprised that that would have survived to 2019. Can you show a testcase?

**Comment by tjhei** [comment]
This is what I get when running explicit.prm from #3119 (1.0 refinement with "strain rate"):
![image](https://user-images.githubusercontent.com/1531285/58756316-5fa25f80-84b3-11e9-8efe-3161eb201109.png)

**Comment by gassmoeller** [comment]
Maybe the error indicator in the last few cells is essentially 0?

**Comment by tjhei** [comment]
> Maybe the error indicator in the last few cells is essentially 0?

likely. I think it would be more intuitive if a refinement fraction of 1.0 does a global refinement, though.

**Comment by gassmoeller** [comment]
I agree. Maybe just check for this special case in the function.

**Comment by bangerth** [comment]
Yes, I bet the error is *exactly* zero in those places. I suspect that it's not an important issue, but you'll have to update deal.II for that.
